vector/src/main/java/im/vector/adapters/RoomViewHolder.java

```diff
@@ -190,10 +191,23 @@ public void populateViews(final Context context,
             vRoomUnreadCount.setVisibility(View.GONE);
         }
 
+        // Customize the cell background when a server notice is unread
+        if (DinumUtilsKt.isServerNotice(room) && roomSummary.mUnreadEventsCount != 0) {
+            itemView.setBackgroundColor(ContextCompat.getColor(context, R.color.tchap_light_blue_color));
+        } else {
+            itemView.setBackgroundResource(0);
```

Comment:
That works in your case.
Since API level 16 you should use `setBackground(null)` to preserve the view's padding.
https://developer.android.com/reference/android/view/View#setBackground(android.graphics.drawable.Drawable)